src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -43,9 +43,7 @@ def __init__(self, *args, **kwargs):
 
     @cached_property
     def client(self):
-        import azul.dss
-        with azul.dss.shared_credentials():
```

Comment:
This was bugging me like crazy. I think it only worked because the AzulTDRClient constructor made a BQ round-trip, otherwise the AuthorizedHttp instance would have been initialized outside of the context manager which would have failed.